Create responsive

Hana-latif98 opened the pull request.

Hana-latif98 committed:
Create responsive by width

Hana-latif98 committed:
Create responive code for height & block browser to overlap elements

Hana-latif98 committed:
Adjust advertisement space top margin >1400px

Hana-latif98 committed:
Set height to avoid elements overlapping when resized

Hana-latif98 committed:
block elements from overlapping - 769-1120

Hana-latif98 committed:
Add height to avoid elements from overlapping >500px

Hana-latif98 committed:
Fix responsive code to avoid elements from overlapping

Hana-latif98 committed:
Add max height to change layout - 769-1120

Hana-latif98 committed:
Remove margin from main function box <500

Hana-latif98 committed:
Add min-height to avoid elements from overlapping <500

Hana-latif98 committed:
Add max-height to stop main function from stretching <500

Hana-latif98 committed:
Fix line width to avoid overflow

Hana-latif98 committed:
Change width of dropdown content when resized

Hana-latif98 committed:
Change commercial holder size >768

Hana-latif98 committed:
Add a few mock ads

Hana-latif98 committed:
Remove one breakpoint

Hana-latif98 committed:
Add hover design on drop down content

Hana-latif98 committed:
Add cursor: pointer

Hana-latif98 committed:
Remove transform scale down on main function box <500

jovove commented:
Content changes:
- Navbar
- Height of the landing page
- Font size of the dropdown menus
![Kapture 2019-11-26 at 8 46 40](https://user-images.githubusercontent.com/39692676/69609212-496ff900-1029-11ea-9985-85945d446228.gif)

jovove commented:
![Kapture 2019-11-26 at 8 49 07](https://user-images.githubusercontent.com/39692676/69609369-a2d82800-1029-11ea-9a13-5c8c2e3704aa.gif)
Input fields changes, size of button changes, the navbar glitches out

jovove commented:
![image](https://user-images.githubusercontent.com/39692676/69609427-c69b6e00-1029-11ea-8ab2-ab1bd329a34a.png)
This is different from the sketches, the headlines and texts align differently

jovove commented:
![image](https://user-images.githubusercontent.com/39692676/69609472-e763c380-1029-11ea-8a41-3a93de370f58.png)
Content differs from design in abstract

jovove commented:
![image](https://user-images.githubusercontent.com/39692676/69609505-fba7c080-1029-11ea-9cd0-c053cc98f246.png)
Designs have different boxes and text styling, which is the correct version?

jovove commented:
![image](https://user-images.githubusercontent.com/39692676/69609541-0eba9080-102a-11ea-9119-a189473a3480.png)
The height in the content is different in your designs compared to your code?

jovove commented on aboveFold.css:
```diff
@@ -20,7 +32,6 @@
     text-align: center;
     max-width: 880px;
-    max-height: 382px;
     padding-top: 40px;
```
You can combine all paddings in one line:
```suggestion
    padding: 40px 0 40px 52px;
```

jovove commented on aboveFold.css:
```diff
@@ -406,9 +424,8 @@
         padding-right: 40px;
         margin-left: auto;
```
All margins can be written as one line
```suggestion
        margin: 0 auto;
```

jovove commented:
Is the `cursor: pointer;` for all the links?

Hana-latif98 commented:
> Is the `cursor: pointer;` for all the links?

I decided to add `cursor: pointer;` to all components that are clickable! Or should I remove it?

Hana-latif98 committed:
Change width of threeIconsFlexContent

Hana-latif98 committed:
Fix 3 Icons spacing and positioning of components

Hana-latif98 committed:
Fix 3 Icons size and positioning of text and button

Hana-latif98 committed:
Fix width of ex flat images to meet in the middle

Hana-latif98 committed:
Add min-height to full scree - 769-1120

Hana-latif98 committed:
Change font size on transformed main function box

Hana-latif98 committed:
Fix button size in main function box when transformed

Hana-latif98 committed:
Change HBW illustration spacing and remove padding

Hana-latif98 committed:
Change text size and fix spacing of 3 Icons components

Hana-latif98 committed:
Change body text size in example Flats

Hana-latif98 committed:
Change main function box size to fit design

Hana-latif98 commented:
> Content changes:
> 
> * Navbar
> * Height of the landing page
> * Font size of the dropdown menus
>   ![Kapture 2019-11-26 at 8 46 40](https://user-images.githubusercontent.com/39692676/69609212-496ff900-1029-11ea-9985-85945d446228.gif)

I changed the height of the content when resized and I also changed the font size in the main function so that should be looking better now!

The navigation bar is coded as intended which is for the logo to be lifted above the text and the create account button moves above the login button. I can change this if it is not living up to a certain standard :)

Hana-latif98 commented:
> ![image](https://user-images.githubusercontent.com/39692676/69609427-c69b6e00-1029-11ea-8ab2-ab1bd329a34a.png)
> This is different from the sketches, the headlines and texts align differently

This is also changed and I added a font-size of 16px to be more readable!

Hana-latif98 commented:
> ![image](https://user-images.githubusercontent.com/39692676/69609472-e763c380-1029-11ea-8a41-3a93de370f58.png)
> Content differs from design in abstract

I changed the code so that the corner of the images touch, however I didn't want the whole row to exceed over 1200px. This means the image is not as wide as the design, I think it looks better this way.

Hana-latif98 committed:
Fix header size >1400px

Hana-latif98 committed:
Adjust MF-Box padding and fullscreen margin - 769-1120

Hana-latif98 committed:
Change size of main function without using transform

Hana-latif98 committed:
Change a break point

jovove commented:
> > Content changes:
> > 
> > * Navbar
> > * Height of the landing page
> > * Font size of the dropdown menus
> >   ![Kapture 2019-11-26 at 8 46 40](https://user-images.githubusercontent.com/39692676/69609212-496ff900-1029-11ea-9985-85945d446228.gif)
> 
> I changed the height of the content when resized and I also changed the font size in the main function so that should be looking better now!
> 
> The navigation bar is coded as intended which is for the logo to be lifted above the text and the create account button moves above the login button. I can change this if it is not living up to a certain standard :)

- I disagree strongly with this layout. It is underwhelming and the amount of negative space makes the content to spread out and out of date.

- The navigation doesn't seem coordinated, you have left aligned, the logo being pushed by margin, a button with background color taking up space and the button with white background looks like regular text and does not create the pattern one might think.

- The advertising have jumped to the top now as well, then the whole concept of the building illustrations feel like they are not being used as their intended purpose and the advertising at the top takes away the focus from the main content.

- There are two different dividers with drop-shadow in place — (also there is no master version of the sketch file? There is only a branch - you need to have a master file for the design to make sure which version is the latest version to be approved).

- The mobile version of the site has a advertising that is odd, is that a approved google ads size? 
![image](https://user-images.githubusercontent.com/39692676/69855257-6ad01f80-128b-11ea-8ea5-75eed1482942.png)

jovove reviewed (changes requested):
Added some notes with some changes — this branch is becoming too large so probably need to fix those issues and open up a new branch to sort it out. The project is currently to large in the files to manage and needs a better structure.

Hana-latif98 committed:
Fix commercial holder and commercials - 769-1130

Hana-latif98 committed:
Remove break point: 1120x & 748y

Hana-latif98 committed:
Fix headerTextHolder to stop wrapping content

Hana-latif98 committed:
Change contect margin to be aligned properly

Hana-latif98 committed:
Align header text to be in the center of the page

Hana-latif98 committed:
Lower mainFunctionBox top margin to fit the screen better

Hana-latif98 committed:
Change fullScreen height to fit more content in screen - 769-1130

Hana-latif98 committed:
Change mobile ad box shadow to be a bit softer

Hana-latif98 committed:
Change size of breaking point

Hana-latif98 committed:
Correct Header content and move what was removed to footer

Hana-latif98 committed:
Move Social Media icons to create space for text - footer

Hana-latif98 committed:
Move header text to footer - <1130px

Hana-latif98 committed:
Make fullscreen a little taller

Hana-latif98 committed:
Create and fill folder Assests

Hana-latif98 committed:
Create and fill Images folder within Assets

Hana-latif98 committed:
Change search path for images

Hana-latif98 committed:
Place all images in folder

Hana-latif98 committed:
Create and fill folder stylesheets

Hana-latif98 committed:
Connect stylesheets and images

Hana-latif98 committed:
Rename assets to be co-ordinated

Hana-latif98 committed:
Rename assets to be coordinated

Hana-latif98 committed:
Clean code and better the structure

Hana-latif98 committed:
Add commercial holder 2 - smaller screens

Hana-latif98 committed:
Add cloud illustration to mobile size

Hana-latif98 committed:
Move Mobile Ad placement to resemble other designs

Hana-latif98 committed:
Move Mobile Ad to resemble other designs

Hana-latif98 committed:
Improve mobile ad positioning

Hana-latif98 committed:
Change positioning of mobile ad to hanging version

Hana-latif98 committed:
Lower margin top on main function box - mobile

Hana-latif98 committed:
Lower margin-top on main function box more - mobile

Hana-latif98 committed:
Change footer breakpoints

Hana-latif98 committed:
Improve footer design and make body text visible

Hana-latif98 committed:
Change footer breakpoint to match other designs

Hana-latif98 committed:
Add breakpoint to improve design small screens - 320px

Hana-latif98 committed:
Change margin-top mfb & change fullscreen height

Hana-latif98 committed:
Change header break point to 1000px instead of 1130px

Hana-latif98 committed:
Remove one div from header that wasn't needed

jovove commented:
![image](https://user-images.githubusercontent.com/39692676/70530109-6d1b5f00-1b52-11ea-9d0b-f188da291269.png)
Links are dissapearing

![image](https://user-images.githubusercontent.com/39692676/70530150-81f7f280-1b52-11ea-9929-7abe8e6464d9.png)
Different links when responsive? 

![Kapture 2019-12-10 at 13 41 08](https://user-images.githubusercontent.com/39692676/70530247-bf5c8000-1b52-11ea-8528-cdf71f55731d.gif)
One advertising just disappears and what was previous 100% height decreases, could you explain your process? 

![image](https://user-images.githubusercontent.com/39692676/70530406-15c9be80-1b53-11ea-84f0-62e0619ca573.png)
Images don't intersect precisely. Please take a look at that and document what might be the cause

![image](https://user-images.githubusercontent.com/39692676/70530530-66d9b280-1b53-11ea-9c7e-de7c1f814a9e.png)
Content does not align

Other:
- The rest of the ads have disappeared completely when going into smaller devices

jovove commented:
![image](https://user-images.githubusercontent.com/39692676/70530823-ff703280-1b53-11ea-8f5b-03590fa70e02.png)

What dimensions are the google ads? the 470x68 size is nowhere to be found on google's recommendations (reference: https://support.google.com/google-ads/answer/7031480?hl=en)

Hana-latif98 committed:
Change links in hamburger menu to be more relative - mobile

Hana-latif98 committed:
Change percentage to make example flat images corners to meet

Hana-latif98 committed:
Same height of above fold - >769

Hana-latif98 committed:
Create better visual balance in section three icons

Hana-latif98 commented:
> ![image](https://user-images.githubusercontent.com/39692676/70530109-6d1b5f00-1b52-11ea-9d0b-f188da291269.png)
> Links are dissapearing
These are meant to be removed and are moved down to the footer. I was struggling to make the space work so I moved it out completely as I have seen other websites do this too.
> 
> ![image](https://user-images.githubusercontent.com/39692676/70530150-81f7f280-1b52-11ea-9929-7abe8e6464d9.png)
> Different links when responsive?
This has been changed so the header links are now in the slider!
> 
> ![Kapture 2019-12-10 at 13 41 08](https://user-images.githubusercontent.com/39692676/70530247-bf5c8000-1b52-11ea-8528-cdf71f55731d.gif)
> One advertising just disappears and what was previous 100% height decreases, could you explain your process?
Height has been changed now to keep the designs cohesive! When it comes to advertising I was struggling to get the adverts of that size into the small space so I decided to remove a few by process of elimination. This is only because I want the main function box to fit in the home screen, maybe I could move the adverts somewhere else, like between partitions (like blog posts do). 
> 
> ![image](https://user-images.githubusercontent.com/39692676/70530406-15c9be80-1b53-11ea-84f0-62e0619ca573.png)
> Images don't intersect precisely. Please take a look at that and document what might be the cause
This should be fixed now, didn't zoom in when checking it so missed the space between!
> 
> ![image](https://user-images.githubusercontent.com/39692676/70530530-66d9b280-1b53-11ea-9c7e-de7c1f814a9e.png)
> Content does not align
I had to change the title to be able to make it align. Tried via sketch to make it work but struggling with this. If it's not preferred to shorten the title then I can take another look at it. Thought I'd give it a shot as it is now. 
> 
> Other:
> 
> * The rest of the ads have disappeared completely when going into smaller devices
This has to do with the screen size and me wanting to prioritise the main function box. Especially in the mobile version!

Hana-latif98 commented:
> ![image](https://user-images.githubusercontent.com/39692676/70530823-ff703280-1b53-11ea-8f5b-03590fa70e02.png)
> 
> What dimensions are the google ads? the 470x68 size is nowhere to be found on google's recommendations (reference: https://support.google.com/google-ads/answer/7031480?hl=en)

The ad itself is 468 x 60 (third from the bottom in the google size add link) but the div holding it is a little larger to create the illusion of a border :)

Hana-latif98 committed:
Add height break point to give components more breathing space

Hana-latif98 committed:
Add favicon

Hana-latif98 committed:
Add title in browser

Hana-latif98 committed:
Improve spacing in mobile version

Hana-latif98 committed:
Add more margin on top f mainfunctionBox - mobile

Hana-latif98 committed:
Improve margin on top of mainfunctionBox - mobile

Hana-latif98 committed:
Fix fullScreen size to match other versions

Hana-latif98 committed:
Improve footer and add more content

Hana-latif98 committed:
Complete footer in all versions
